litmus/director/tcid-iuoi14-openebs-resource-limit-installation/test.yml

```diff
@@ -134,6 +103,27 @@
           set_fact:
             openebsid: "{{ get_openebs.json.id }}"
 
+        - name: Get the node1_id
+          set_fact:
+            node1_id: "{{ node_id[0].split('n')[1] }}"
+
+        - name: Get the node2_id
+          set_fact:
+            node2_id: "{{ node_id[1].split('n')[1] }}"
+
+        - name: labeling nodes with new approach
```

Comment:
Rename the task to - `Labeling the nodes of the cluster`

Reply:
Also is the labeling done post the openebses POST request in the new approach?

Reply:
Yes here we have to label nodes after giving POST request in openebses because we need openebses Id to label nodes.